File: Solution_Java/Week01/26-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        if (nums.length == 0){
            return 0;
        }
        if (nums.length==1){
            return 1;
        }
        int i=0;
        int j=i+1;
        while (j<nums.length){
            if (nums[i]!=nums[j]){
                nums[++i] = nums[j];
            }
            j+=1;

        }
        return i+1;
    }
}

File: Solution_Go/Week01/26-remove-duplicates-from-sorted-array.go
func removeDuplicates(nums []int) int {
    if len(nums) == 0 {
        return 0
    }
    if len(nums) == 1 {
        return 1
    }

    i := 0
    j := 1
    for j<len(nums) {
        if nums[i]!=nums[j] {
            i += 1
            nums[ i ] = nums[ j ]
        }
        j++
    }

    return i+1
}
